### PR TITLE
Improve description of `GetNodeOrDefault` in C# API

### LINKED ISSUE #6577: Mistake on generic GetNodeOrNull() description.
**Your Godot version:** v4.0.beta10.mono.official.d0398f62f

**Issue description:** little mistake in description of `T Node.GetNodeOrNull<T>(NodePath path) where T : class` method, no error is logged.

**URL to the documentation page:** I found the error in the help while coding, I don't know where it is in the documentation, the description of the non generic method is okay.

This is my first issue :)

### PR DESCRIPTION
**Pull request by guilleatm**
Closes https://github.com/godotengine/godot-docs/issues/6577.

### CODE REVIEW

**Comment by guilleatm** [comment]
PR from issue in docs: https://github.com/godotengine/godot-docs/issues/6577

Provided same text as GDScript version: https://docs.godotengine.org/en/latest/classes/class_node.html#class-node-method-get-node-or-null

First PR :)

**Comment by YuriSizov** [comment]
Note that you will need to squash your commits into one, with a clear but short message, before this can be merged. You can do it later, in case there are some review comments that you need to address, but you will have to do this eventually. For a reference when crafting a good commit message you can read [this](https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#format-your-commit-messages-with-readability-in-mind).

**Comment by guilleatm** [comment]
Squash done

File: modules/mono/glue/GodotSharp/GodotSharp/Core/Extensions/NodeExtensions.cs [hunk 1]
```diff
@@ -48,13 +48,7 @@ public partial class Node
...
 
         /// <summary>
-        /// Fetches a node. The <see cref="NodePath"/> can be either a relative path (from
-        /// the current node) or an absolute path (in the scene tree) to a node. If the path
-        /// does not exist, a <see langword="null"/> instance is returned and an error
-        /// is logged. Attempts to access methods on the return value will result in an
-        /// "Attempt to call &lt;method&gt; on a null instance." error.
-        /// Note: Fetching absolute paths only works when the node is inside the scene tree
-        /// (see <see cref="IsInsideTree"/>).
+		///	Similar to <see cref="GetNode"/>, but does not log an error if <paramref name="path"/> does not point to a valid <see cref="Node"/>.
```
Comment (raulsntos) [inline modules/mono/glue/GodotSharp/GodotSharp/Core/Extensions/NodeExtensions.cs:51]: Use spaces for indentation. Also, I think the line is a bit long and would be better to break it up.

```suggestion
        /// Similar to <see cref="GetNode"/>, but does not log an error if <paramref name="path"/>
        /// does not point to a valid <see cref="Node"/>.
```

**Review by raulsntos — CHANGES_REQUESTED** [review]

**Review by raulsntos — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks! And congratulations on getting your first PR merged :)

**Comment by guilleatm** [comment]
Thanks! :)

**Pull request merged**